pages/03.program/program.md

```diff
@@ -9,6 +9,7 @@ process:
     markdown: true
     twig: true
 visible: true
+tepplate: sessions
```

Comment:
hmmm, looks like a typo here... seems to not be affecting the site. **Suggest removing this line**.